bindata/ocpdeployment/metallb.yaml

```diff
@@ -0,0 +1,211 @@
+apiVersion: policy/v1beta1
```

Comment:
No need to have this here which is supposed to be generic.
This can live just on the ocp counterpart (and it will never conflict because it's a different path)

Reply:
the ocp manifest is removed now